packages/replay-next/components/inspector/KeyValueRenderer.tsx

```diff
@@ -164,7 +166,11 @@ export default function KeyValueRenderer({
       {before}
       {name != null ? (
         <>
-          <span className={nameClass} data-test-name="KeyValue-Header">
+          <span
+            className={nameClass}
+            data-getter-value={isGetterValue}
```

Comment:
This will simplify the CSS since it won't result in a `data-getter-value` attribute on a non-getter value.
```suggestion
            data-getter-value={isGetterValue || undefined}
```